src/Decorations.ts

```diff
@@ -18,16 +19,16 @@ export default class Decorations {
   decorations: NamedDecoration[];
   decorationMap: DecorationMap;
 
-  constructor() {
-    const fontWidthPx = 8.04;
+  constructor(fontSize: FontSize) {
+    console.log(fontSize)
```

Comment:
We'll prob want to either remove or migrate to `console.debug` before merging into master

Reply:
Gonna remove most logs